be/src/getPartnerToken.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Function that verifies that the incoming request is authorised to act on
+ * behalf of a partner and provides a partner token in response.
+ */
+export type GetPartnerToken = (
+  request: GetPartnerTokenRequest,
+) => Promise<string>;
```

Comment:
Change from `Promise<string | undefined>` is breaking, but we don’t have a release yet 🙂